Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
@davorbonaci A Rubyist may be more opinionated, but AFAIK these are always *supposed* to be in the root directory of the app per the [spec](http://bundler.io/man/gemfile.5.html)